www-apps/sonarr/files/sonarr.init

```diff
@@ -0,0 +1,8 @@
+#!/sbin/openrc-run
+
+pidfile="/run/${RC_SVCNAME}.pid"
+command="$(command -v mono)"
```

Comment:
openrc will search for the command in PATH, so the command -v call is unnecessary.